packages/vue-components/src/Dropdown.vue

```diff
@@ -230,7 +230,10 @@ export default {
     }
 
     .navbar .dropdown-toggle {
-        color: inherit;
         text-decoration: none;
     }
+
+    li {
```

Comment:
Is this still necessary? Can you show screenshots to illustrate the effect of this?

Reply:
Yup still necessary!

This is without the padding 
![image](https://user-images.githubusercontent.com/77204346/229011070-a5fc0df1-51a2-47bd-bcf4-032de6f5a208.png)

This is with the padding 
![image](https://user-images.githubusercontent.com/77204346/229011169-ece90768-b8e6-452a-864c-75231e63827e.png)